cli/dcoscli/cluster/main.py

```diff
@@ -254,7 +254,10 @@ def _user_cert_validation(cert_str):
     msg = "SHA256 fingerprint of cluster certificate bundle:\n{}".format(
             pp_fingerprint)
 
-    return confirm(msg, False)
+    if not confirm(msg, False):
+        raise DCOSException("Couldn't get confirmation for the fingerprint.")
```

Comment:
i think this is breaking the intension of this function
```
Prompt user for validation of certification from cluster
    :param cert_str: cluster certificate bundle
    :type cert_str: str
    :returns whether or not user validated cert
    :rtype: bool
```

the function should return True or False, i think you should probably raise the exception in the caller

Reply:
I'll take a look at it 👍